### ISSUE TITLE
ARIA state or property is permitted - Passed examples 10 and 11 fail with ARIA 1.2

### ISSUE THREAD

**Issue by dd8** [act-rules/act-rules.github.io#1370]
https://act-rules.github.io/rules/5c01ea#passed-example-10
https://act-rules.github.io/rules/5c01ea#passed-example-11

Passed examples 10 and 11 are (technical) passes with ARIA 1.1 but failures with ARIA 1.2, because `aria-expanded` is no longer allowed on `role=complementary`

https://www.w3.org/TR/wai-aria-1.1/#complementary
https://www.w3.org/TR/wai-aria-1.2/#complementary

Allowing `aria-expanded` on `role=complementary` in ARIA 1.1 was a spec bug so these aren't good pass examples:
https://github.com/w3c/aria/issues/681

**Comment by Jym77** [comment]
🤔  I guess a `<button>` with `aria-pressed` should do the trick.
